README.md

```diff
@@ -1,89 +1,96 @@
+API.
+
+## Documentation
+
+See the [Python API docs](https://stripe.com/docs/api/python#intro).
 
 ## Installation
 
-You don't need this source code unless you want to modify the
-package. If you just want to use the Stripe Python bindings, you
-should run:
+You don't need this source code unless you want to modify the gem. If you just
+want to use the package, just run:
 
     pip install --upgrade stripe
 
 or
 
     easy_install --upgrade stripe
 
-See http://www.pip-installer.org/en/latest/index.html for instructions
-on installing pip. If you are on a system with easy_install but not
-pip, you can use easy_install instead. If you're not using virtualenv,
-you may have to prefix those commands with `sudo`. You can learn more
-about virtualenv at http://www.virtualenv.org/
-
-To install from source, run:
+Install from source with:
 
     python setup.py install
 
-## Documentation
+### Requirements
+
+* Python 2.6+ or Python 3.3+ (PyPy supported)
 
-Please see https://stripe.com/docs/api/python for the most up-to-date documentation.
-
-## Logging
-
-There are a few ways to get some insight into what requests and responses the client is making and getting back from the Stripe API.
-The python client can be told to emit either `debug` or `info` logs.
-`debug` will give more verbose information, and `info` will be more compact, but omit things like request/response bodies.
-
-You can enable logging in a few ways:
-    1. Set the environment variable `STRIPE_LOG` to the value `debug` or to `info`
-       ```
-       $ export STRIPE_LOG=debug
-       ```
-    2. set `stripe.log` to `'debug'` or `'info'`
-       ```py
-       import stripe
-       stripe.log = 'debug'
-       ```
-    3. Set up python logging and set the logging level to the level you desire
-       ```py
-       import logging
-       logging.basicConfig()
-       logging.getLogger('stripe').setLevel(logging.DEBUG)
-       ```
-
-If you are running code in production, it's preferable to set up [python logging explicitly](https://docs.python.org/2/library/logging.html). This will give you control over filtering logs, and where the logs are output. Using the `STRIPE_LOG` and `stripe.log` methods will always result in logs going to standard out, which you may or may not want.
-
-Note that setting `stripe.log` will supersede the setting of `STRIPE_LOG`, but it can be turned off by setting it back to `None`. Example:
-
-```py
-import os
+## Usage
+
+The library needs to be configured with your account's secret key which is
+available in your [Stripe Dashboard][api-keys]. Set `stripe.api_key` to its
+value:
+
+``` python
 import stripe
+stripe.api_key = "sk_test_..."
+
+# list charges
+stripe.Charge.list()
 
-print os.environ['STRIPE_LOG']  # => 'info'
-stripe.log = 'debug'  # debug logs will also print now
-stripe.log = None  # now only info logs will print
+# retrieve single charge
+stripe.Charge.retrieve("ch_1A2PUG2eZvKYlo2C4Rej1B9d")
 ```
 
```

Comment:
Should we add a "Per-request configuration" paragraph here like in stripe-ruby's README?